index.js

```diff
@@ -236,6 +239,10 @@ function getStsClient(region) {
   });
 }
 
+function isRetryableError(error) {
+  return error instanceof Error && error.code && (error.code === 'IDPCommunicationErrorException' || error.code === 'InvalidIdentityToken');
```

Comment:
Can it be simplified to this?

```js
return error instanceof Error && ['IDPCommunicationErrorException', 'InvalidIdentityToken'].includes(error?.code);
```

Reply:
This is nice

Reply:
@miluoshi But build fails with
```
You may need an additional loader to handle the result of these loaders.
| 
| function isRetryableError(error) {
>   return error instanceof Error && ['IDPCommunicationErrorException', 'InvalidIdentityToken'].includes(error?.code);
| }

```

Reply:
So probably just `return error instanceof Error && ['IDPCommunicationErrorException', 'InvalidIdentityToken'].includes(error.code);` should work